lib/SetAPI/differentialexpressionmatrix/DifferentialExpressionMatrixSetInterfaceV1.py

```diff
@@ -90,10 +90,9 @@ def _check_get_differential_expression_matrix_set_params(self, params):
             raise ValueError(
                 '"ref" parameter field specifiying the DifferentialExpressionMatrix set is required'
             )
-        elif not check_reference(params["ref"]):
+        if not check_reference(params["ref"]):
             raise ValueError('"ref" parameter must be a valid workspace reference')
-        if "include_item_info" in params:
-            if params["include_item_info"] not in [0, 1]:
-                raise ValueError(
-                    '"include_item_info" parameter field can only be set to 0 or 1'
-                )
+        if "include_item_info" in params and params["include_item_info"] not in [0, 1]:
```

Comment:
combine conditions

Reply:
Depending on how time goes, I may put in standard tests for all the modules that have these checks. Most of these *Set classes could be made simple subclasses of a generic set class, which would get rid of a lot of boilerplate.